Tools/gdb/libpython.py

```diff
@@ -1076,7 +1080,10 @@ def _f_builtins(self):
         return self._f_special("f_builtins")
 
     def _f_code(self):
-        return self._f_special("f_code", PyCodeObjectPtr.from_pyobject_ptr)
+        return self._f_special("f_executable", PyCodeObjectPtr.from_pyobject_ptr)
+
+    def _f_executable(self):
+        return self._f_special("f_executable", PyObjectPtr.from_pyobject_ptr)
```

Comment:
No need to repeat the default converter:
```suggestion
        return self._f_special("f_executable")
```